### PR TITLE
OHARA-1212 Remove "cluster" APIs

### PR DESCRIPTION
**Pull request by chia7712**

### CODE REVIEW

**Comment by konekoya** [comment]
Hmm, @chia7712 there are a few more minor tweaks that I'd like to make in this PR. Is it okay that I can pick this up and continue the work?

File: ohara-manager/build.gradle [hunk 1]
```diff
@@ -64,21 +64,17 @@ task runManager {
     exec {
       commandLine 'yarn'
```
Comment (konekoya) [inline ohara-manager/build.gradle]: Let's remove this command as the `yarn setup` script will take care of everything that we need (dep installation and creating a fresh build of UI)

:)

**Comment by chia7712** [comment]
> @chia7712 there are a few more minor tweaks that I'd like to make in this PR. Is it okay that I can pick this up and continue the work?

sure. Please take over this PR. thanks!

**Comment by konekoya** [comment]
@chia7712 I've made some more changes. Please review them :)

**Comment by chia7712** [comment]
> I've made some more changes. Please review them :)

+1

**Comment by konekoya** [comment]
retry manager

**Comment by konekoya** [comment]
Thanks @chia7712 @jackyoh :)

**Pull request merged**